features/flexbox.yml.dist

```diff
@@ -3,14 +3,14 @@
 
 status:
   baseline: high
-  baseline_low_date: 2020-09-22
-  baseline_high_date: 2023-03-22
+  baseline_low_date: 2020-01-15
```

Comment:
This change isn't because of https://github.com/mdn/browser-compat-data/pull/22597. 2020-01-15 is the release date of Edge 79. I think we need to tweak this feature to align with https://caniuse.com/flexbox.

Reply:
I've sent https://github.com/web-platform-dx/web-features/pull/1228.

Reply:
Also https://github.com/mdn/browser-compat-data/pull/23338 since Firefox Android 52 shouldn't be showing up here.